pkg/trace/testutil/otlp.go

```diff
@@ -87,6 +97,16 @@ func setOTLPSpan(span ptrace.Span, s *OTLPSpan) {
 		insertAttributes(ev.Attributes(), e.Attributes)
 		ev.SetDroppedAttributesCount(e.Dropped)
 	}
+	links := span.Links()
+	for _, l := range s.Links {
+		li := links.AppendEmpty()
+		bytes, _ := hex.DecodeString(l.TraceID)
+		li.SetTraceID(*(*pcommon.TraceID)(bytes))
+		bytes, _ = hex.DecodeString(l.SpanID)
+		li.SetSpanID(*(*pcommon.SpanID)(bytes))
+		insertAttributes(li.Attributes(), l.Attributes)
+		li.SetDroppedAttributesCount(l.Dropped)
```

Comment:
I think we only set dropped attributes count when it's > 0?

Reply:
This code is initializing the OTel pdata that will be translated in the test so setting it explicitly isn't a problem (although it defaults to 0 so we could special-case it).

Similarly for attributes.

Reply:
Alright, makes sense.